src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -82,6 +85,7 @@ export default {
   },
   computed: {
     isGroupMarker: ({ item: { type } }) => type === TopicTypes.groupMarker,
+    isDeprecated: ({ item: { deprecated } }) => !!deprecated,
```

Comment:
We should open a PR on Swift-DocC to document this new optional `deprecated` property for the [`Node`](https://github.com/apple/swift-docc/blob/c9e274ec87b293613c6da2d58e2c10bcd83945db/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json#L32) Open API specification before merging this.

Reply:
We'll also need to implement support in Swift-DocC for emitting this. We can do both at the same time.

Reply:
> We should open a PR on Swift-DocC to document this new optional deprecated property for the [Node](https://github.com/apple/swift-docc/blob/c9e274ec87b293613c6da2d58e2c10bcd83945db/Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json#L32) Open API specification before merging this.
> We'll also need to implement support in Swift-DocC for emitting this. We can do both at the same time.

This work is being tracked here rdar://89363995. Thanks!